addons/web_editor/static/src/js/editor/add_snippet_dialog.js

```diff
@@ -174,7 +174,7 @@ export class AddSnippetDialog extends Component {
             }
             clonedSnippetEl.classList.remove("oe_snippet_body");
             const snippetPreviewWrapEl = document.createElement("div");
-            snippetPreviewWrapEl.classList.add("o_snippet_preview_wrap", "position-relative", "fade");
+            snippetPreviewWrapEl.classList.add("o_snippet_preview_wrap", "position-relative", "d-none");
```

Comment:
> This happened because the code that places the "tour pointer" on the
element checks if the element is visible after every mutation. When the
preview of the block (where the user must click) appears, it has a
"fade" class, which makes it invisible until it gets the "show" class.
Because of this, the tour code thought the element wasn’t visible and
didn’t add the pointer.

Weird explanation: if it checks after each mutation, then it should detect the "show" class addition mutation and it should work no? Anyways, if you are not sure, I am ok with fixing it that way (not the first issue we have with mutations in the new tour system), we can check with the framework team later.

Reply:
> Weird explanation: if it checks after each mutation, then it should detect the "show" class addition mutation and it should work no? 

no because at the moment the "show" is added, the element is still hidden. There is a small transition...
I'm looking to fix it in another way (by changing the `trigger` of the click step in the `instertSnippet` function)

Reply:
> I'm looking to fix it in another way

In the end, I left it as is